### ISSUE TITLE
[Feature Request] Security Reminder

### ISSUE THREAD

**Issue by BillyCroan** [khpylon/MachEWidget#44]
The data you get from Ford should tell you if the car is parked, on or off, and if the doors are locked windows are open, etc. And the vehicle's location.

Wouldn't it be nice if my phone reminded me that I left my car in an unsecured state?

If the windows or doors are open, or doors left unlocked while parked and ignition off, start a timer, of length specified by user.  When that timer runs out, refresh data.  If vehicle is still parked, off, and not secured, send a notification and play an audible alarm on the phone.

One problem:
The app doesn't refresh data all the time and cannot lest it get blocked by Ford.  We need some way to become aware that the car has been parked quickly without blasting ford 24/7.

Solution:
Use the phones location awareness services.
If the app senses that the phone is beginning to move at car speeds, then request one data refresh of the car.  If the car is on and moving also, then monitor speed of phone until it stops moving and request another refresher car data.  If the phone and cars stop at the same time then a refresh the Ford data once a minute max 5 times until it shows the car ignition is now off, and the phone GPS is no longer zipping around at car speeds.

Optionally, allow user to specify a parking location where parking unsecured is okay. For instance I usually keep my windows open when I park in my locked garage.  In my lifestyle if I am parking my car within 100 m of my home then it's guaranteed to be in my garage.  I know some people don't always park inside versus outside so they might not use this exception feature.

**Comment by khpylon** [comment]
This is an interesting idea.  My first impression is it would be quite a bit of coding.  I'd also need to figure how to handle owners who have multiple vehicles.
